### DISCUSSION TITLE
We know Turf.js accepts WGS84, but what is the EPSG?

### DISCUSSION THREAD

**Discussion by neelduttahere** [Turfjs/turf#2704] [General]
https://gis.stackexchange.com/questions/342056/do-leaflet-and-turfjs-have-different-projections#:~:text=1-,Turf.,two%20types%3A%20vector%20or%20raster.

Based on the above post, it mentions:

> uses unprojected WGS84 coordinate system, which is EPSG:6326

I could not verify this in the official documentation of turf.js, Would love some clarity regarding this.

**Comment by twelch** [comment]
It is EPSG:4326 - https://epsg.io/4326
